x-pack/plugins/serverless_search/public/application/components/connectors_overview.tsx

```diff
@@ -118,7 +121,39 @@ export const ConnectorsOverview = () => {
       <EuiPageTemplate.Section restrictWidth color="subdued">
         <ConnectorPrivilegesCallout />
         {connectorsLoading || (data?.connectors || []).length > 0 ? (
-          <ConnectorsTable />
+          <>
+            {showCallOut && (
+              <>
+                <EuiCallOut
+                  size="m"
+                  title="Coming soon Elastic managed connectors."
+                  iconType="pin"
+                  onDismiss={onDismiss}
+                >
+                  <p>
+                    {i18n.translate(
+                      'xpack.serverlessSearch.connectorsOverview.calloutDescription',
+                      {
+                        defaultMessage:
+                          "We're actively developing Elastic managed connectors, that won't require any self-managed infrastructure. You'll be able to handle all configuration in the UI. This will simplify syncing your data into a serverless Elasticsearch project.",
+                      }
+                    )}
+                  </p>
+                  <EuiButton
+                    data-test-subj="serverlessSearchConnectorsOverviewLinkButtonButton"
```

Comment:
@sphilipse or @TattdCodeMonkey could you confirm that using proper `data-test-subj` values for each interactive item is the only thing we should take care of in order to track the use cases listed below for this Coming Soon pages?

- Users that load the Elastic managed connectors coming soon page
- Users that load the Elastic managed connectors coming soon page + Click on Submit form
- Users that click in the Callout to know more

As far as I know we can track those in _Fullstory_ and _kibana-usage-counters_. Is it correct? If so,  is there any related guideline?